mlflow/langchain/runnables.py

```diff
@@ -269,10 +292,15 @@ def _save_runnable_with_steps(steps, file_path: Union[Path, str], loader_fn=None
     steps_path = save_path / _STEPS_FOLDER_NAME
     steps_path.mkdir()
 
+    steps = model.steps
     if isinstance(steps, list):
         generator = enumerate(steps)
     elif isinstance(steps, dict):
         generator = steps.items()
+    else:
+        raise MlflowException(
+            f"steps for must be either a list or a dictionary. Got {type(steps).__name__}."
```

Comment:
nit: Can we elaborate this error message a bit more? It seems `steps` is kinda internal terminology which doesn't show up in their doc frequently, so it might be a bit hard to tell what's wrong with the chain from just seeing "steps". Maybe it can be like "Runnable xxx should have `steps` attribute with a list or dictionary. Got ..."?